trade-finance-manager-ui/server/controllers/case/amendments/amendCoverEndDate.test.js

```diff
@@ -0,0 +1,127 @@
+import { isTfmFacilityEndDateFeatureFlagEnabled, TEAM_IDS } from '@ukef/dtfs2-common';
+import api from '../../../api';
+import { mockRes } from '../../../test-mocks';
```

Comment:
I've added tests for the cover end date controller (the page that comes before the 'is using facility end date' page) - previously there were no tests, so added in a test file to check it redirects users to the correct page on pressing 'Continue'